ocaml/typing/typecore.ml

```diff
@@ -793,6 +799,10 @@ type module_variable =
     mv_uid: Uid.t
   }
 
+type exhaustivity_constraint =
```

Comment:
Let's comment this one too.

Reply:
Done!

Reply:
Thanks. Can we also use "partial" language here instead of "exhaustive"?

Reply:
Yep, done!